pkg/driver/bacnet/merge/emergency.go

```diff
@@ -21,12 +21,15 @@ import (
 	"github.com/vanti-dev/sc-bos/pkg/task"
 )
 
-// AlarmConfig allows configuring a specific bacnet point to raise an Emergency if the
-// value read from that point is anything other than OkValue
+// AlarmConfig allows configuring a specific bacnet point to raise an Emergency if either:
+// - value read from that point is anything other than OkValue
+// - value read from that point is less than GreaterThanOkValue
```

Comment:
Fyi lists in go docs should [have some blank space at the start](https://tip.golang.org/doc/comment#lists)